cost/budget_v_actual/README.md

```diff
@@ -0,0 +1,59 @@
+# Monthly Actual v. Budgeted Spend Report
+
+**As a best practice, this policy should only be applied to the Master Account, and not to each individual RightScale Account. The monthly budget inputs are compared against the current month's actual cloud costs of the applied policy. Reapply this policy on an annual basis to reflect the current year's monthly budgeted cloud costs.**
```

Comment:
This sentence isn't needed.  As a best practice, this policy should only be applied to the Master Account, and not to each individual RightScale Account.
Instead add the new tenancy field.  tenancy "single"
Move the rest of the bolded detail into Functional Details